src/bacnet/basic/object/netport.c

```diff
@@ -2665,6 +2669,88 @@ bool Network_Port_IPv6_DHCP_Server_Set(
     return status;
 }
 
+/**
+ * For a given object instance-number, gets the IPv6_DHCP_Lease_Time.
+ * Note: depends on Network_Type being set to PORT_TYPE_BIP6 for this object
+ *
+ * @param  object_instance - object-instance number of the object
+ *
+ * @return IPv6_DHCP_Lease_Time
+ */
+uint32_t Network_Port_IPv6_DHCP_Lease_Time(uint32_t object_instance)
+{
+    const unsigned index = Network_Port_Instance_To_Index(object_instance);
+    uint32_t value = 0;
+
+    if (index < BACNET_NETWORK_PORTS_MAX) {
+        if (Object_List[index].Network_Type == PORT_TYPE_BIP6) {
```

Comment:
can be put in the same if statement 
    if ((index < BACNET_NETWORK_PORTS_MAX) &&  (Object_List[index].Network_Type == PORT_TYPE_BIP6) )